q2_feature_table/plugin_setup.py

```diff
@@ -101,6 +131,14 @@
     parameters={},
     outputs=[
         ('merged_data', FeatureData[Taxonomy])],
+    input_descriptions={
+        'data1': 'The first collection of feature taxonomies to be merged.',
+        'data2': 'The second collection of feature taxonomies to be merged.',
+    },
+    parameter_descriptions={},
+    output_descriptions={
+        'merged_data': 'The resulting collection of feature taxonomies.'
```

Comment:
*The resulting collection of feature taxonomies containing all feature taxonomies from `data1` and `data2`.*